### PR TITLE
Added possibility to connect one genisys instance to many synapse/nemisys proxies

### PR DESCRIPTION
**Pull request by Tarik02**
### Description

This PR allows to connect one genisys to multiple synapse or nemisys instances.

### Reason to modify

I'm want to use more than one proxy for my server network, because one proxy cannot handle very many players.

### Tests & Reviews

I have tested the code and it works. This not breaks old configs.

### CODE REVIEW

**Comment by ishitatsuyuki** [comment]
1. Your formatting is messed up
2. Squash your commits
3. Multiple entrance for user doesn't make sense
-1

File: src/pocketmine/resources/genisys_chs.yml [hunk 1]
```diff
@@ -97,18 +97,32 @@ redstone:
  #设置脉冲频率, 默认: 1s
  pulse-frequency: 1
 
+#TODO: Translate
 synapse:
- #如果你想使用 Synapse API 或连接到一个synapse服务器,请将enabled置为true
+ #If you want to use Synapse API or connect to a synapse server, enable this
```
Comment (dktapps) [inline src/pocketmine/resources/genisys_chs.yml]: What's the point of doing this?

Comment (robske110) [inline src/pocketmine/resources/genisys_chs.yml]: i think Chinese->English...

Comment (dktapps) [inline src/pocketmine/resources/genisys_chs.yml]: Yes, in a CHINESE configuration file.

Comment (robske110) [inline src/pocketmine/resources/genisys_chs.yml]: Lol

Comment (Tarik02) [inline src/pocketmine/resources/genisys_chs.yml]: I'm not know chinese, but i'm want to update this config.

**Comment by ishitatsuyuki** [comment]
I don't know what you're trying to do with this, but it doesn't make sense. You're also modifying things that shouldn't be changed. Closing.

**Pull request closed without merging**